dcos/dcos-ui
# Fix message catalog

## Testing
1. Try to start a community package from the Catalog.
2. Verify that the confirmation looks the same as before.
3. The stop job run modal content should be still translated in Chinese.

## Trade-offs
None.

## Dependencies
None.

## Screenshots
None.
